### ISSUE TITLE
Reduction axis 0 is empty in shape [0]

### ISSUE THREAD

**Issue by ghazalee70** [balancap/SSD-Tensorflow#103]
I have a problem with testing my trained network (on my own data). The Jaccard output is empty and therefore the argmax does not work. I faced no errors while training the network and the training loss converged. I have no idea how to debug this issue.

**Comment by ypymy** [comment]
did you solve this problem? i meet the similar issue ...

**Comment by charan223** [comment]
I also have this problem
solution guys?

**Comment by Cuky88** [comment]
Could you solve this?
In [Issue #66](https://github.com/balancap/SSD-Tensorflow/issues/66) @villanuevab said, it was due to her preprocessing, but I have the same issue without any custom preprocessing.

**Comment by kritchie** [comment]
I had the same issue. My problem was that I was using the 512x512 network with 640x480 images. So that could be one cause for errors. Check your image sizes ! Hope it can help some people.

**Comment by South2North** [comment]
I solve similar problem by removing the images without bbox and label in TFRecord files.
